cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1002,6 +1002,7 @@ class DnsSchema(BaseSchema):
     """Represent the schema of Dns Settings."""
 
     disable_managed_dns = fields.Bool(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
+    hosted_zone_id = fields.Str(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
```

Comment:
Let's add an integ test for this new setting.

Reply:
Added